dev_scripts/env.py

```diff
@@ -17,9 +17,11 @@
 DEFAULT_SHOW_DOCKERFILE = False
 DEFAULT_DOWNLOAD_PYSIDE6 = False
 
+PYSIDE6_VERSION ="6.6.3.1"
 PYSIDE6_RPM = "python3-pyside6-{pyside6_version}-1.fc{fedora_version}.x86_64.rpm"
 PYSIDE6_URL = (
-    "https://packages.freedom.press/yum-tools-prod/dangerzone/f{fedora_version}/%s"
+    #"https://packages.freedom.press/yum-tools-prod/dangerzone/f{fedora_version}/%s"
```

Comment:
Nitpick (caught also by [lint CI](https://app.circleci.com/pipelines/github/freedomofpress/dangerzone/2489/workflows/2530c259-d269-4a61-97cc-1dbf4e1c3617/jobs/32917?invite=true#step-103-522_86)).
```suggestion
    # "https://packages.freedom.press/yum-tools-prod/dangerzone/f{fedora_version}/%s"
```

Is the previous URL commented-out temporarily? If so, some context on why/when it can be uncommented would be useful as a comment.

Reply:
Yeap, it will be resolved in the parent PR. I'll resolve this conversation, as a similar comment exists in that PR.